praw/models/reddit/submission.py

```diff
@@ -519,7 +520,8 @@ def shortlink(self) -> str:
         """Return a shortlink to the submission.
 
         For example http://redd.it/eorhm is a shortlink for
-        https://www.reddit.com/r/announcements/comments/eorhm/reddit_30_less_typing/.
+        https://www.reddit.com/r/announcements/comments/eorhm/
```

Comment:
Comments that are longer than the line length are fine. This makes the docstring much less readable.